dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/domain/TrackerDto.java

```diff
@@ -33,7 +33,10 @@
  * @author Morten Svanæs <msvanaes@dhis2.org>
  */
 public interface TrackerDto {
+
   String getUid();
 
   TrackerType getTrackerType();
+
+  int getIndex();
```

Comment:
I wonder if we should give this a more descriptive name 🤔? As the index is actually the index into the import "bundle" or so.